### ISSUE TITLE
yamlFiles property does not work with tj-actions/changed-files action

### ISSUE THREAD

**Issue by ccentilli** [thiagodonferreira/yaml-schema-checker#10]
The README says that the yaml file can intake a "list of files to be validated". I tried using the output from [tj-actions/changed-files](https://github.com/marketplace/actions/changed-files) as the input for yamlFiles, but the test returns a false positive saying that it found 0 files. I think it would be useful to make these two actions work together since tj-actions/changed-files is so common. 
<img width="1293" alt="image" src="https://user-images.githubusercontent.com/105660278/178996230-6c2f8db3-2ca0-453e-afeb-27741603e918.png">

**Comment by thiagodnf** [comment]
Hi @ccentilli 

I have released a new version v0.0.8 that tries to fix this error;

```yaml
uses: thiagodnf/yaml-schema-checker@v0.0.8
with:
  jsonSchemaFile: schemas/example.schema.json
  yamlFiles: |
    file_1.yml,file_2.yml # two files in the same line
    file_3.yml
    folder/subfolder/**/*.yml
```

Using `tj-actions/changed-files`, it should be:

```yaml

- name: Get changed files
  id: changed-files
  uses: tj-actions/changed-files@v23.1
  with:
    separator: ","

 - uses: thiagodnf/yaml-schema-checker@v0.0.8
   with:
     jsonSchemaFile: schemas/example.schema.json
     yamlFiles: ${{ steps.changed-files.outputs.all_changed_files }}
```

Please do not forget `separator: ","` for tj-actions/changed-files

Please let me know if now it is working.

Thank you

**Comment by ccentilli** [comment]
Hello! I just ran some tests and the issue has been fixed. Thank you so much!
